packages/core/src/browser/shell/tab-bar-toolbar.tsx

```diff
@@ -404,6 +404,9 @@ export class TabBarToolbarRegistry implements FrontendApplicationContribution {
      * By default returns with all items where the command is enabled and `item.isVisible` is `true`.
      */
     visibleItems(widget: Widget): Array<TabBarToolbarItem | ReactTabBarToolbarItem> {
+        if (widget.isDisposed) {
```

Comment:
fo instance in the side bar, the title is still visible, but associated widget is disposed since it is collapsed, with bad timing it can throw in own contributions